flaml/searcher/flow2.py

```diff
@@ -124,15 +124,14 @@ def _init_search(self):
             if callable(getattr(domain, 'get_sampler', None)):
                 self._tunable_keys.append(key)
                 sampler = domain.get_sampler()
-                # if isinstance(sampler, sample.Quantized):
-                #     sampler_inner = sampler.get_sampler()
-                #     if str(sampler_inner) == 'Uniform':
-                #         self._step_lb = min(
-                #             self._step_lb, sampler.q/(domain.upper-domain.lower))
-                # elif isinstance(domain, sample.Integer) and str(
-                #     sampler) == 'Uniform':
-                #     self._step_lb = min(
-                #         self._step_lb, 1.0/(domain.upper-domain.lower))
+                if isinstance(sampler, sample.Quantized):
+                    sampler_inner = sampler.get_sampler()
+                    if str(sampler_inner) == 'Uniform':
```

Comment:
why only handling 'Uniform'? What about 'LogUniform'? 

and why in step_lower_bound(), only 'LogUniform' is handled?